src/vue-components/inflections.vue

```diff
@@ -182,8 +189,10 @@
       },
       forms: function () {
         let forms = []
-        if (this.selectedView && this.selectedView.forms) {
+        if (this.selectedView && this.selectedView.forms && this.selectedView.forms.values.length > 0) {
```

Comment:
Can we get rid of this method here now? I think it isn't used anymore correct?

Reply:
Oh, sorry :) I have missed it. Done.